src/js/views/maps/LayerDetailsView.js

```diff
@@ -295,9 +292,8 @@ define([
           return this;
         } catch (error) {
           console.log(
```

Comment:
I have seen these console.log statements after catch all over the code, so I did not remove any in this file. @robyngit do we want to keep them?

Reply:
@alonakos yes, good question!!

Most of these were my doing back in the day!! We used to have an issue where tiny mistakes in one part of the codebase would crash the entire app. So we decided to use more `try-catch`es to catch these errors, but I was over zealous. While we should certainly still catch errors to handle them gracefully, it's not needed all over the place like it is now. Also, we should do something more useful than `console.log`ing the error statement.

Also, since we're moving toward the AirBnB style guide, we want to avoiding using `console.log` in production code. So I would say remove the `try-catch`es and `console.log`s when they are redundant or at least not very useful. When it makes sense, we should keep the `try-catch` and add a useful fallback behaviour in the `catch` part.

Reply:
Sounds good! I opened a new issue here: https://github.com/NCEAS/metacatui/issues/2500